frontend/src/metabase/selectors/settings.ts

```diff
@@ -40,3 +40,12 @@ const getUtmSource = (features: TokenFeatures) => {
     return features.hosting ? "starter" : "oss";
   }
 };
+
+const getTokenFeatures = (state: State) => getSetting(state, "token-features");
+
+export const getIsPaidPlan = createSelector(
```

Comment:
I don't see tests for this utility. Do they exist?

Reply:
Good catch. They don't, since I just created this selector in this PR. I will add one.

Reply:
Added tests